keras_cv/models/object_detection/yolox/__internal__/layers/yolox_decoder.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import tensorflow as tf
+from tensorflow import keras
+
+import keras_cv.layers as cv_layers
+from keras_cv import bounding_box
+
+
+class DecodePredictions(keras.layers.Layer):
+    """Decodes the predictions from YoloX head.
+
+    This layer is similar to the decoding code in `YoloX.compute_losses`. This is
```

Comment:
Add description on what this layer does, in case users don't read YoloX.compute_losses